File: src/java/SimulationObjects/AgentAttributes.java
package SimulationObjects;

/*
 * created by: b farooq, poly montreal
 * on: 22 october, 2013
 * last edited by: b farooq, poly montreal
 * on: 22 october, 2013
 * summary: 
 * comments:
 */

public class AgentAttributes
{
    public static class AgentType
    {
        public static int Household = 0;
        public static int Person = 1;
    }

    public static class HouseholdSize
    {
        public static int SingleMale = 0;
        public static int SingleFemale = 1;
        public static int TwoPersons = 2;
        public static int ThreePersons = 3;
        public static int FourPersons = 4;
        public static int FiveNPlusPersons = 5;
    }

    public static class DwellingType
    {
        public static int Separate = 0;
        public static int SemiAttached = 1;
        public static int Attached = 2;
        public static int Apartments = 3;
    }

    public static class NumOfCars
    {
        public static int NoCar = 0;
        public static int OneCar = 1;
        public static int TwoCars = 2;
        public static int ThreeOrMore = 3;
    }

    public static class NumOfWorkers
    {
        public static int None = 0;
        public static int One = 1;
        public static int TwoOrMore = 2;
    }

    public static class NumOfKids
    {
        public static int None = 0;
        public static int One = 1;
        public static int TwoOrMore = 2;
    }

    public static class NumWithUnivDeg
    {
        public static int None = 0;
        public static int One = 1;
        public static int TwoOrMore = 2;
    }

    public static class IncomeLevel
    {
        public static int ThirtyOrLess = 0;
        public static int ThirtyToSevetyFive = 1;
        public static int SeventyFiveToOneTwentyFive = 2;
        public static int OneTwentyFiveToTwoHundred = 3;
        public static int TwohundredOrMore = 4;
    }

    /// <summary>
    /// Person attributes
    /// </summary>
    public static class HouseholdType
    {
        public static int SinglePersonHhld = 0;
        public static int FamilyHhld = 1;
        public static int NonFamilyHhld = 2;
        public static int Collectivehhld = 3;
    }

    public static class HouseholdSize2
    {
        public static int SinglePerson = 0;
        public static int TwoPersons = 1;
        public static int ThreePersons = 2;
        public static int FourPersons = 3;
        public static int FivePersons = 4;
        public static int SixNPlusPersons = 5;
    }

    public static class Sex
    {
        public static int Male = 0;
        public static int Female = 1;
    }

    public static class MaritalStatus
    {
        public static int Single = 0;
        public static int Married = 1;
        public static int Widowed = 2;
        public static int Divorced = 3;
    }

    public static class EducationLevel
    {
        public static int none = 0;
        public static int primary = 1;
        public static int secondary = 2;
        public static int tertiary = 3;
    }

    public static class EmploymentStatus
    {
        public static int Employed = 0;
        public static int Unemployed = 1;
        public static int Inactive = 2;
        public static int UnderFifteen = 3;
    }

    public static class Age
    {
        public static int LessThanFifteen = 0;
        public static int FifteenToTwentyFour = 1;
        public static int TwentyFiveToThirtyFour = 2;
        public static int ThirtyFiveToFortyFour = 3;
        public static int FortyFiveToFiftyFour = 4;
        public static int FiftyFiveToSixtyFour = 5;
        public static int SixtyFiveToSeventyFour = 6;
        public static int SeventyFiveAndMore = 7;
    }
}

File: src/c#/SimulationObjects/AgentAttributes.cs
/*
 * created by: b farooq, poly montreal
 * on: 22 october, 2013
 * last edited by: b farooq, poly montreal
 * on: 22 october, 2013
 * summary: 
 * comments:
 */

namespace SimulationObjects
{
    public enum AgentType
    {
        Household = 0,
        Person = 1
    }

    /// <summary>
    /// Household attributes
    /// </summary>

    public enum HouseholdSize
    {
        SingleMale = 0,
        SingleFemale = 1,
        TwoPersons = 2,
        ThreePersons = 3,
        FourPersons = 4,
        FiveNPlusPersons = 5
    };

    public enum DwellingType
    {
        Separate = 0,
        SemiAttached = 1,
        Attached = 2,
        Apartments = 3
    };

    public enum NumOfCars
    {
        NoCar = 0,
        OneCar = 1,
        TwoCars = 2,
        ThreeOrMore = 3
    };

    public enum NumOfWorkers
    {
        None = 0,
        One = 1,
        TwoOrMore = 2
    };

    public enum NumOfKids
    {
        None = 0,
        One = 1,
        TwoOrMore = 2
    };

    public enum NumWithUnivDeg
    {
        None = 0,
        One = 1,
        TwoOrMore = 2
    }

    public enum IncomeLevel
    {
        ThirtyOrLess = 0,
        ThirtyToSevetyFive = 1,
        SeventyFiveToOneTwentyFive = 2,
        OneTwentyFiveToTwoHundred = 3,
        TwohundredOrMore = 4
    }

    /// <summary>
    /// Person attributes
    /// </summary>

    public enum HouseholdType
    {
        SinglePersonHhld = 0,
        FamilyHhld = 1,
        NonFamilyHhld = 2,
        Collectivehhld = 3
    };

    public enum HouseholdSize2
    {
        SinglePerson = 0,
        TwoPersons = 1,
        ThreePersons = 2,
        FourPersons = 3,
        FivePersons = 4,
        SixNPlusPersons = 5
    };

    public enum Sex
    {
        Male = 0,
        Female = 1
    };

    public enum MaritalStatus
    {
        Single = 0,
        Married = 1,
        Widowed = 2,
        Divorced = 3
    };

    public enum EducationLevel
    {
        none = 0,
        primary = 1,
        secondary = 2,
        tertiary = 3
    };

    public enum EmploymentStatus
    {
        Employed = 0,
        Unemployed = 1,
        Inactive = 2,
        UnderFifteen = 3
    };

    public enum Age
    {
        LessThanFifteen = 0,
        FifteenToTwentyFour = 1,
        TwentyFiveToThirtyFour = 2,
        ThirtyFiveToFortyFour = 3,
        FortyFiveToFiftyFour = 4,
        FiftyFiveToSixtyFour = 5,
        SixtyFiveToSeventyFour = 6,
        SeventyFiveAndMore = 7
    };
}